telegram/ext/filters.py

```diff
@@ -970,165 +986,160 @@ def filter(self, message: Message) -> bool:
     """Messages that contain :attr:`telegram.Message.voice_chat_scheduled`.
 
     .. versionadded:: 13.5
     """
 
     class _VoiceChatStarted(MessageFilter):
         __slots__ = ()
-        name = 'filters.StatusUpdate.VOICE_CHAT_STARTED'
 
         def filter(self, message: Message) -> bool:
             return bool(message.voice_chat_started)
 
-    VOICE_CHAT_STARTED = _VoiceChatStarted()
+    VOICE_CHAT_STARTED = _VoiceChatStarted(name="filters.StatusUpdate.VOICE_CHAT_STARTED")
     """Messages that contain :attr:`telegram.Message.voice_chat_started`.
 
     .. versionadded:: 13.4
     """
 
     class _VoiceChatEnded(MessageFilter):
         __slots__ = ()
-        name = 'filters.StatusUpdate.VOICE_CHAT_ENDED'
 
         def filter(self, message: Message) -> bool:
             return bool(message.voice_chat_ended)
 
-    VOICE_CHAT_ENDED = _VoiceChatEnded()
+    VOICE_CHAT_ENDED = _VoiceChatEnded(name="filters.StatusUpdate.VOICE_CHAT_ENDED")
     """Messages that contain :attr:`telegram.Message.voice_chat_ended`.
 
     .. versionadded:: 13.4
     """
 
     class _VoiceChatParticipantsInvited(MessageFilter):
         __slots__ = ()
-        name = 'filters.StatusUpdate.VOICE_CHAT_PARTICIPANTS_INVITED'
 
         def filter(self, message: Message) -> bool:
             return bool(message.voice_chat_participants_invited)
 
-    VOICE_CHAT_PARTICIPANTS_INVITED = _VoiceChatParticipantsInvited()
+    VOICE_CHAT_PARTICIPANTS_INVITED = _VoiceChatParticipantsInvited(
+        "filters.StatusUpdate.VOICE_CHAT_PARTICIPANTS_INVITED"
+    )
     """Messages that contain :attr:`telegram.Message.voice_chat_participants_invited`.
 
     .. versionadded:: 13.4
     """
 
-    name = 'filters.STATUS_UPDATE'
-
-    def filter(self, update: Update) -> bool:
-        return bool(
-            self.NEW_CHAT_MEMBERS.check_update(update)
-            or self.LEFT_CHAT_MEMBER.check_update(update)
-            or self.NEW_CHAT_TITLE.check_update(update)
-            or self.NEW_CHAT_PHOTO.check_update(update)
-            or self.DELETE_CHAT_PHOTO.check_update(update)
-            or self.CHAT_CREATED.check_update(update)
-            or self.MESSAGE_AUTO_DELETE_TIMER_CHANGED.check_update(update)
-            or self.MIGRATE.check_update(update)
-            or self.PINNED_MESSAGE.check_update(update)
-            or self.CONNECTED_WEBSITE.check_update(update)
-            or self.PROXIMITY_ALERT_TRIGGERED.check_update(update)
-            or self.VOICE_CHAT_SCHEDULED.check_update(update)
-            or self.VOICE_CHAT_STARTED.check_update(update)
-            or self.VOICE_CHAT_ENDED.check_update(update)
-            or self.VOICE_CHAT_PARTICIPANTS_INVITED.check_update(update)
-        )
+    class _All(UpdateFilter):
+        __slots__ = ()
 
+        def filter(self, update: Update) -> bool:
+            return bool(
+                StatusUpdate.NEW_CHAT_MEMBERS.check_update(update)
```

Comment:
looking at this wall of `or` I realize again how helpfult it would be if TG just had a special `status_update` flag/type xD